pkg/exporters/stackdriver/config/config.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"time"
+
+	"k8s.io/node-problem-detector/pkg/exporters/stackdriver/gce"
+)
+
+var (
+	defaultExportPeriod          = (60 * time.Second).String()
+	defaultEndpoint              = "monitoring.googleapis.com:443"
+	defaultMetadataFetchTimeout  = (600 * time.Second).String()
+	defaultMetadataFetchInterval = (10 * time.Second).String()
+)
+
+type StackdriverExporterConfig struct {
+	ExportPeriod                string       `json:"exportPeriod"`
+	APIEndpoint                 string       `json:"apiEndpoint"`
+	Metadata                    gce.Metadata `json:"gceMetadata"`
+	MetadataFetchTimeout        string       `json:"metadataFetchTimeout"`
+	MetadataFetchInterval       string       `json:"metadataFetchInterval"`
+	PanicOnMetadataFetchFailure bool         `json:"panicOnMetadataFetchFailure"`
```

Comment:
Shall we put the default values of the above 3 config options in the config file, so that users know this can be configured?

Reply:
SG :) done.